src/main/java/liquibase/ext/bigquery/database/BigQueryDatabase.java

```diff
@@ -148,10 +117,14 @@ public String getDefaultDriver(String url) {
 
         return null;
     }
+    @Override
+    public boolean supportsDatabaseChangeLogHistory() {
+        return true;
+    }
 
     @Override
     public boolean supportsSequences() {
-        return this.supports(Sequence.class);
```

Comment:
I think this one could still invoke supports method as it may be deprecated soon (or it is already)